docs/cli.adoc

```diff
@@ -0,0 +1,182 @@
+:showtitle:
+:toc: left
+:icons: font
+
+= Oxide CLI
+
+The `oxide` CLI is used to access Nexus' external API, which is
+the public interface to Omicron.
+
+For more detail, refer to https://docs.oxide.computer/cli/manual[`oxide`'s CLI Manual].
+
+== Quick demo
+
+The `oxide` CLI can be installed from the https://github.com/oxidecomputer/cli/releases[latest release].
+
+Run `oxide auth login` to authenticate with your Oxide account. Alternatively,
+`oxide` will respect the `OXIDE_TOKEN` and `OXIDE_HOST` environment variables.
+
+If you find bugs or have feedback, leave it on the https://github.com/oxidecomputer/cli/issues[cli repo].
+
+Here's a small demo that creates a project, creates an instance, and attaches a disk to it:
+
+[source,text]
+----
+$ oxide org create myorg \
+    --description "My organization"
+✔ Created organization myorg
+
+$ oxide project create myproject \
+    --description "My project" \
+    --organization myorg
+✔ Created project myorg/myproject
+
+$ oxide instance create myinstance \
+    --description "My instance" \
+    --project myproject \
+    --org myorg \
+    --hostname "myinstance.maze-war.com" \
+	  --ncpus 1 \
+	  --memory 8
+✔ Created instance myinstance in myorg/myproject
+
+$ oxide instance view myinstance \
+    --org myorg \
+    --project myproject \
+    --format json
+{
+  "id": "99ad2514-050c-4493-9cb9-d9ceba980a98",
+  "name": "myinstance",
+  "description": "My instance",
+  "timeCreated": "2021-11-17T01:45:07.606749Z",
+  "timeModified": "2021-11-17T01:45:07.606749Z",
+  "projectId": "c197b9d2-285c-4e9f-9461-1815ef093c8d",
+  "ncpus": 1,
+  "memory": 8,
+  "hostname": "myinstance.maze-war.com",
+  "runState": "running",
+  "timeRunStateUpdated": "2021-11-17T01:45:09.120652Z"
+}
+
+$ oxide disk create nginx \
+	-D "The nginx disk." \
+	-o myorg \
+	-p myproject \
+	--size 10
+✔ Created disk nginx in myorg/myproject
+
+
+$ oxide disk view nginx \
+    --org myorg \
+    --project myproject \
+    --format json
+{
+  "id": "551bbe67-3640-41c9-b968-249a136e5e31",
+  "name": "nginx",
+  "description": "The nginx disk.",
+  "timeCreated": "2021-11-17T01:47:36.524136Z",
+  "timeModified": "2021-11-17T01:47:36.524136Z",
+  "projectId": "c197b9d2-285c-4e9f-9461-1815ef093c8d",
+  "snapshotId": null,
+  "size": 1024,
+  "state": {
+    "state": "detached"
+  },
+  "devicePath": "/mnt/nginx"
+}
+
+$ oxide disk attach nginx myinstance \
+	-o maze-war \
+	-p prod-online
+✔ Attached disk nginx to instance myinstance in myorg/myproject
+
+$ oxide instance disks myinstance \
+	-o maze-war \
+	-p prod-online \
+  --format json
+{
+  "instanceId": "99ad2514-050c-4493-9cb9-d9ceba980a98",
+  "diskId": "551bbe67-3640-41c9-b968-249a136e5e31",
+  "diskName": "nginx",
+  "diskState": {
+    "state": "attached",
+    "instance": "99ad2514-050c-4493-9cb9-d9ceba980a98"
+  }
+}
+
+# Alternatively, you can use the API command to run any endpoint.
+# This operates like a fancy, authenticated curl.
```

Comment:
Could this be pulled out of the code block? It gets a little hard to read among the help message text below.

Reply:
Sure thing, done